src/App.jsx

```diff
@@ -0,0 +1,26 @@
+import {
+	Home,
+	Cart,
+	Wishlist,
+	Products,
+	ProductDetails,
+	Profile,
+	Authentication,
+} from "./pages/index";
```

Comment:
`/index` is not required. If the directory has index.js then it directly attempts import from index

Reply:
Thanks @ravikumar0403 will make that chnage